doc/operations/platforms/snp.rst

```diff
@@ -0,0 +1,28 @@
+AMD SEV-SNP
+===================
+
+.. warning:: SEV-SNP support is currently experimental and in active development.
+
+How to use the AMD SEV-SNP platform
+-----------------------------------
+CCF must run on an AMD CPU which supports SEV-SNP.
+
+To use SNP, ``virtual`` must be specified in ``COMPILE_TARGETS``, at compile time. If any other targets are specified, you may also need to specify the ``TEST_ENCLAVE=virtual`` runtime argument.
+
+Attestation
+-----------
+SNP attestation provide several fields needed to establish trust. This is specific to the SNP hardware as deployed in confidential Azure Container Instances, see `here <https://learn.microsoft.com/en-us/azure/confidential-computing/confidential-containers>`_ for more information.
+
+- Measurement
+
+Measurement is a digest of the UVM (Utility Virtual Machine) that is running, which CCF stores as a :doc:`code id <../../audit/builtin_maps>`. New nodes joining a network will provide their code id and the primary will perform an identity check against the table entries.
```

Comment:
```suggestion
Measurement is a digest of the initial memory pages for the SEV-SNP VM, which CCF stores as a :doc:`code id <../../audit/builtin_maps>`. New nodes joining a network will provide their code id and the primary will perform an identity check against the table entries.
```

See https://www.amd.com/system/files/TechDocs/SEV-SNP-strengthening-vm-isolation-with-integrity-protection-and-more.pdf page 17.

UVM is an ACI-only concept.

Reply:
Addressed both comments about what measurement means in https://github.com/microsoft/CCF/pull/4330/commits/5414843c99f4f9b082d5b470c4409c05422ff009